medusa/subtitles.py

```diff
@@ -211,17 +215,18 @@ def from_ietf_code(code, unknown='und'):
         return Language(unknown) if unknown else None
 
 
-def from_country_letter_to_name(country):
-    """Return the country name for the given 2 letter country.
+def from_country_code_to_name(code):
+    """Convert a 2 letter country code to a country name.
 
-    :param code: the 2 letter language code
+    :param code: the 2 letter country code
     :type code: str
     :return: the country name
     :rtype: str
     """
-    country_name = babelfish.Country(country).name if country and country in babelfish.COUNTRIES else ''
-
-    return country_name
+    try:
+        return Country(code.upper()).name
+    except ValueError:
+        return
```

Comment:
@fernandog 
It should be faster since it only looks up the value once, instead of twice (if in babelfish.COUNTRIES + Country(code)), it looks cleaner (at least to me) and it's a more pythonic approach.